### PR TITLE
Some minor improvements

### PR DESCRIPTION
**Pull request by jessealama**

### CODE REVIEW

**Comment by jessealama** [comment]
The aim here is to remove or downplay references to a new *built-in data type* in favor of talking about new functionality (which may not take the form of a new built-in type).

**Comment by ljharb** [comment]
Doesn’t have to block the PR, but i think i did voice in the last plenary discussion that the proposal only holds its weight imo if it is a new primitive type.

**Comment by jessealama** [comment]
> Doesn’t have to block the PR, but i think i did voice in the last plenary discussion that the proposal only holds its weight imo if it is a new primitive type.

I feel similarly, though I still think there's some value in what we've got so far. I've just added new "Future work" section to the README that explains how we intend to keep the door open to that brighter future.

File: README.md [hunk 1]
```diff
@@ -18,22 +16,20 @@ This whole proposal is a big open question, and we’d welcome your participatio
...
+What’s the issue? Why aren’t JS Numbers good enough? In what sense are they not “exact”? How is it possible that JavaScript's Numbers get something wrong, and have been getting it wrong for so long?
 
-As currently defined in JavaScript, Numbers 64-bit binary floating-point numbers, and the conversion from most decimal values to binary floats rarely is an exact match. For instance: 0.5 works, but 0.1 does not (the "0.1" 64-bit floating point number is actually 0.1000000000000000055511151231257827021181583404541015625). Same for 0.2, 0.3, … Statistically, most human-authored decimal numbers cannot be exactly represented as a binary float.
+As currently defined in JavaScript, Numbers 64-bit binary floating-point numbers. The conversion from most decimal values to binary floats rarely is an exact match. For instance: the decimal number 0.5 can be exactly represented in binary, but 0.1 does not; in fact, the "0.1" 64-bit floating point number is actually 0.1000000000000000055511151231257827021181583404541015625. Same for 0.2, 0.3, … Statistically, most human-authored decimal numbers cannot be exactly represented as a binary floating-point number (AKA float).
```
Comment (jirkamarsik) [inline README.md:21]: Minor: the decimal number 0.5 can be ..., but 0.1 *cannot*;

File: README.md [hunk 2]
```diff
@@ -255,6 +251,37 @@ The library of numerical functions here is deliberately minimal. It is based aro
...
+
+However, the door is not *permanently* closed to overloading. It is just that the bar for adding it to JS is very high. We may be able to meet that bar if we get enough positive developer feedback and work with implementors to find a path forward.
+
+### Decimal literals
+
+In earlier discussions of this proposal, we had advocated for adding new decimal literals to the language: `1.289m` (notice the little `m` suffix). Indeed, since decimals are numbers—essentially, basic data akin to the existing binary floating-point numbers—it is quite reasonable to aim for giving them their own "space" in the syntax.
+
+However, as with operator overloading, we have received strong implementor feedback that this is very unlikely to happen.
+
+Nonetheless, we are working on making sure that the v1 version of the proposal, sketched here, is compatible with a future in which decimal literals exist. As with operator overloading, discussions with JS engine implementors needs to be kept open to find out what can be done to add this feature. (On the assumption that a v1 of decimals exists, one can add support for literals fairly straightforwardly using a Babel transform.)
```
Comment (jirkamarsik) [inline README.md:273]: Typo: discussions with JS engine implementors *need* to be kept open

File: README.md (hunk 1 above)
Comment (jessealama) [reply README.md]: Good catch! I just pushed a fix.

File: README.md (hunk 2 above)
Comment (jessealama) [reply README.md]: good spot -- fixed!

**Pull request merged**